brewerdigital-llc/VivoCare
# Task/pagination

# Description
Pagination

# Closes issue(s)
Pagination

# Steps to Test
1. Sign in as Donna on actual device (Donna has more than 100 messages sent/received 
2. Go to messages - see messages
3. Send a message
4. Go to appointments - see appointments
5. Sign out and back in as Kristin
6. See dashboard
7. See vitals
8. See other dashboard items
9. Navigate to "view all" screens for all dashboard items
10. Go to/see billing screen
11. Book appointment - see that it appears in the list

# Changes include

# Checklist
